go.mod

```diff
@@ -18,6 +18,7 @@ require (
 	github.com/kr/pretty v0.1.0 // indirect
 	github.com/minio/sha256-simd v0.1.1 // indirect
 	github.com/mitchellh/mapstructure v1.1.2
+	github.com/mr-tron/base58 v1.1.3
```

Comment:
we already have a base58 import.
"github.com/btcsuite/btcutil/base58"

Reply:
fixed